lib/ansible/module_utils/network/meraki/meraki.py

```diff
@@ -152,6 +153,17 @@ def is_update_required(self, original, proposed, optional_ignore=None):
                     is_changed = True
         return is_changed
 
+    def construct_params_list(self, keys, aliases=None):
+        args_list = []
+        for key in keys:
+            item = (aliases[key], self.module.params[key])
+            args_list.append(item)
+        return args_list
+
+    def encode_url_params(self, params):
```

Comment:
I see this function twice in this commit?

Reply:
I had it in there and rewrote it thinking it wasn't there. Fixed.